superset/proxy/api.py

```diff
@@ -182,14 +192,24 @@ def get_ipstring(self, ip_string: str, **_kwargs: Any) -> Response:
         current logged in user, and will then send a request to Alfred to see
         if the passed in ip_string is in any reports/incidents
 
-        :param ip_string: String value representing the ip passed in from the front-end
+        :param ip_string: String value representing the ip(s) passed in from the front-end
         :param _kwargs: Array representing any other arguments passed to the function
         :returns: Response generated from passing values to the make_alfred_connection function
         """
+        user_ips = ip_string.split(",")
+        user_ip_string = ''
+
+        if len(user_ips) > 1:
```

Comment:
Looks like another spot we can adjust the check here and drop the elif.

Reply:
Good catch, completely forgot that the for loop won't run if len(user_emails) == 1